new-site/src/docs/components/side-navigation/tabs.mdx

```diff
@@ -20,6 +20,10 @@ import PageTabs from '../../../components/PageTabs';
   A side navigation is an additional navigation element that is located at the left-hand side of the viewport. It provides an extra level of navigation and also can be helpful in situations where the traditional top navigation feels too clumsy. 
 </LeadParagraph>
 
+<Notification label="A visual rework is coming soon!" className="siteNotification">
+  The HDS team will be unifying the SideNavigation component with the Hel.fi project side navigation during 2022.
```

Comment:
Should we omit the year and use some more vague term, for example, "in the near future"?